Packages/io.chainsafe.web3-unity/Runtime/Scripts/Connection/ReownConnectionProvider.cs

```diff
@@ -21,7 +21,7 @@ namespace ChainSafe.Gaming.UnityPackage.Connection
     /// Reown connection provider used for connecting to a wallet using Reown.
     /// </summary>
     [CreateAssetMenu(menuName = "ChainSafe/Connection Provider/Reown", fileName = nameof(ReownConnectionProvider))]
-    public class ReownConnectionProvider : ConnectionProvider, IReownConfig, IConnectionHandlerProvider
+    public partial class ReownConnectionProvider : ConnectionProvider, IReownConfig, IConnectionHandlerProvider
```

Comment:
so partial worked for extending across assemblies??

Reply:
Ah no, that's left unintentionally